src/librustc_mir/interpret/terminator.rs

```diff
@@ -370,7 +370,7 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                         self.stack.pop();
                         Err(err)
                     }
-                    Ok(v) => Ok(v),
+                    Ok(_) => Ok(()),
```

Comment:
```suggestion
                    Ok(()) => Ok(()),
```

Just to make sure we don't do the wrong thing in the future if the return type changes. Mostly a minor concern though, I just don't really like seeing `Ok(_)` when it's just `Ok(())`.